guide/english/javascript/es6/spread-operator/index.md

```diff
@@ -18,6 +18,15 @@ const arr_2 = [5, 6, 7, 8]
 const arr_final = [...arr_1, ...arr_2]
 // arr_final = [1, 2, 3, 4, 5, 6, 7, 8]
 ```
+Spread operator can be use to copy an Array. Assigment operator makes a reference to the same memory location and any change in one Array affect the others. 

```

Comment:
> Spread operator can be use to copy an Array. Assigment operator makes a reference to the same memory location and any change in one Array affect the others.

You have a typo and some grammar issues with this sentence.  Try changing it to:
```
The spread operator can be used to copy an Array. 
```
Not sure why you are mentioning the Assignment operator here at all.